src/_data/toc/sales.yml

```diff
@@ -255,6 +255,8 @@ pages:
       url: "/payment/zero-subtotal-checkout.html"
   - label: Fraud Protection
     url: "/sales/fraud-protection.html"
+  - label: Braintree
```

Comment:
If this page is already in a toc file, it cannot be used a second time (causes problems with nav switching)